src/test/java/uk/gov/companieshouse/AppealApplicationTests.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.companieshouse;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Map;
+
+import static org.assertj.core.api.BDDAssertions.then;
+
+/**
+ * Basic integration tests
```

Comment:
I think it makes sense to remove the comment. I do not see what value it adds. I'd rather have great class names than documentation that becomes tech debt minute after it was added.